src/firefly/js/ui/CatalogSearchMethodType.css

```diff
@@ -0,0 +1,32 @@
+div.insearch {
+    height: 75px;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: flex-end;
+}
+div.intarget {
+    height: 75px;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+}
+div.size {
+    margin-top: 20px;
+    margin-bottom: 20px;
+    margin-left: 30%;
+    border-bottom: 2px solid #c8c8c8;
+    height: 60px;
+}
+
+div.sizellipse {
```

Comment:
Could use some word separators in this css file.  I've been using the camelCase style.  i.e.  sizeEllipse
Hypens are also popular, but I don't use it because I use '--' for modifier separation as in BEM.